components/03-modules/range/Range.vue

```diff
@@ -0,0 +1,261 @@
+        -
+      </span>
+      <label
+        class="
+            label
+            range-filter__label
+        "
+        :for="upperInput.field.id"
+      >
+        {{ upperInput.field.label }}
+      </label>
+      <input
+        :id="upperInput.field.id"
+        v-model.number.lazy="maxInput"
+        class="input__field range-filter__input range-filter__input--upper"
+        :label="upperInput.label"
+        type="text"
+        :name="upperInput.field.name"
+        :placeholder="upperInput.field.placeholder"
+      >
+      <span
+        v-if="upperInput.suffix"
+        class="range-filter__suffix"
+      >
+        {{ upperInput.suffix }}
+      </span>
+    </div>
+    <alpaca-button
+      v-if="applyLabel"
+      type="button"
+      class="button--secondary button--fluid range-filter__button"
+      :aria-label="applyLabel"
+      @click="apply"
+    >
+      {{ applyLabel }}
+    </alpaca-button>
+  </div>
+</template>
+
+<script>
+  import vueSlider from 'vue-slider-component'
+  import AlpacaButton from '../../02-elements/button/Button'
+
+  export default {
+    components: {
+      vueSlider,
+      AlpacaButton
+    },
+    model: {
+      prop: 'value',
+      event: 'input'
+    },
+    props: {
+      value: {
+        type: Array,
+        required: true
+      },
+      min: {
+        type: Number,
+        required: true
+      },
+      max: {
+        type: Number,
+        required: true
+      },
+      slider: {
+        type: Object,
+        default: () => {
+          return {
+            dotSize: 24,
+            height: 8,
+            tooltip: null,
+            bgStyle: {
+              "backgroundColor": "#73739c"
+            },
+            processStyle: {
+              "backgroundColor": "#31e37d",
+              "box-shadow": "inset 0 0 3px rgba(51, 51, 51, 0.45)"
+            },
+            sliderStyle: {
+              "boxShadow": "0 2px 4px 0 rgba(51, 51, 51, 0.5)"
+            }
+          }
+        }
+      },
+      lowerInput: {
+        type: Object,
+        default: () => {
+          return {
+            label: 'Minimum value',
+            field: {
+              id: 'range-filter-min',
+              name: 'range-filter-min',
+              placeholder: '',
+            },
+            suffix: '$'
+          }
+        }
+      },
+      upperInput: {
+        type: Object,
+        default: () => {
+          return {
+            label: 'Maximum value',
+            field: {
+              id: 'range-filter-max',
+              name: 'range-filter-max',
+              placeholder: '',
+            },
+            suffix: '$'
+          }
+        }
+      },
+      applyLabel: {
+        type: String,
+        default: 'Apply'
+      }
+    },
+    data() {
+      return {
+        currentValue: this.value
+      }
+    },
+    computed: {
+      minInput: {
+        get: function() {
+          return this.currentValue[0]
+        },
+        set: function(minValue) {
```

Comment:
Please, compare behavior with https://alpaca-components.now.sh/components/detail/range.html
It's a little strange, that we can move the min value to the max and then it becomes the max :D